src/main/java/uk/gov/companieshouse/efs/api/email/config/PaymentReportEmailConfig.java

```diff
@@ -29,6 +30,14 @@ public void setScottishEmailAddress(String scottishEmailAddress) {
         this.scottishEmailAddress = scottishEmailAddress;
     }
 
+    public String getSpecialCapitalEmailAddress() {
+        return specialCapitalEmailAddress;
+    }
+
+    public void setSpecialCapitalEmailAddress(final String specialCapitalEmailAddress) {
```

Comment:
Please add unit test coverage

Reply:
this one was easy - added `setGetSpecialCapitalEmailAddress()`